### DISCUSSION TITLE
[Question]: Refresh logs me out. Is it expected?

### DISCUSSION THREAD

**Discussion by thekoc** [danny-avila/LibreChat#2136] [Q&A]
### What is your question?

When I refresh the page, I'm logged out. Is it a expected behavior or a bug?

### More Details

When I refresh the page, I'm logged out. Then I have to enter my username and password again.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vladiliescu** [comment]
Yeah, from what I've seen it's by design -- you can get around this by running it with `npm run backend:dev`

**Comment by thekoc** [comment]
> Yeah, from what I've seen it's by design -- you can get around this by running it with `npm run backend:dev`

Does this disable the authentication completely? I'm expecting the login status to be saved for a while (like x days) as like the "remember me" option on other websites.

**Comment by vladiliescu** [comment]
No, you still need to log in -- feel free to try this.

**Comment by danny-avila** [comment]
You need to setup HTTPS to disable this behavior. It's a security feature: https://github.com/danny-avila/LibreChat/discussions/572
